spec/_widget/components/search.spec.js

```diff
@@ -198,7 +202,7 @@ describe('Search', () => {
         'Forgot Password': 1
       },
       'email not working': {
-        'What\'s an MX Record?': 4,
+        'What\'s an MX Record?': 5,
```

Comment:
If you're wondering how I come up with these out-of-nowhere numbers, when I run the tests (`yarn test`), sometimes they fail:

<img width="924" alt="image" src="https://github.com/user-attachments/assets/248aeaba-b724-4d98-baa2-4a4540fe8862" />

And we also have a warning that indicates that we can actually expect the article in the results higher than it is:

<img width="930" alt="image" src="https://github.com/user-attachments/assets/e1367097-170b-4570-8985-d848b4dbab23" />